include/wolfboot/wolfboot.h

```diff
@@ -219,13 +219,15 @@ extern "C" {
 #ifndef WOLFBOOT_FLAGS_INVERT
 #define IMG_STATE_NEW       0xFF
 #define IMG_STATE_UPDATING  0x70
+#define IMG_STATE_FINAL_SWAP  0xA0
```

Comment:
if FINAL_SWAP is after UPDATING it should have a value where zeros are added to 0x70 for normal inverted flash allowing multiple writes. This value peraps should be 0x30

Reply:
I see what you mean, NEW 0b1111 -> UPDATING 0b0111 -> FINAL_SWAP 0b0011